ci/build/build-standalone-release.sh

```diff
@@ -2,6 +2,7 @@
 set -euo pipefail
 
 # This is due to an upstream issue with RHEL7/CentOS 7 comptability with node-argon2
+# TODO@jsjoeio see if we can remove
```

Comment:
❓ This is legacy code. I'm not sure if we still need to tell npm to build from source 🤔

cc @yisibl any ideas?

Reply:
Theoretically not needed, node-rs/argon2 already compiles the binary (.node) file for CentOS.